### DISCUSSION TITLE
Tutorial on creating a unique homepage that differs from page 2+

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#63] [Topic Ideas]
Recently, an [issue came up on the support forums](https://wordpress.org/support/topic/front-page-74/?view=all) where a user wanted to create a custom-designed homepage with a separate design for page 2+.  In classic themes, this is merely a few simple bits of PHP thrown in to contextually change the output.  However, block themes are static and have no such option.  The forum thread does have a solution.

While this was a user request, I could see this being something that could work for the Developer Blog.  Designers, freelancers, and agencies might want to employ the solution in their own work.  It is not an uncommon pattern.

CC: @kathrynwp, @richtabor - Adding both of you since you were involved with figuring this out.

**Comment by DaisyOlsen** [comment]
I think this is a cool idea that can showcase how to do something interesting with a Block Theme (or just the Site Editor) I see that it hasn't been officially approved to move forward, but I would like to tackle this for my next post.

FWIW, several of the classic themes I used to support back in the day had a different layout on pages 2+ like this.

**Reply by bph** [reply]
Would you consider broaden it a bit considering that is a similar topic under discussion and coordinate with @justintadlock @annezazu 

- #66

**Reply by DaisyOlsen** [reply]
I think I see how it might be possible to combine the two topics but I actually think they both can easily stand alone without being duplicates. My preference would be to add this post as an exploration into an interesting and specific technique to accomplish something that might not be obvious.

**Reply by justintadlock** [reply]
I'd like to see more feedback from @annezazu on the other ticket before considering potentially merging them or broadening this one.  I really think this idea is large enough to warrant its own post, particularly as a how-to or step-by-step tutorial on creating this type of homepage.  

I could also see many of these types of real-world scenarios as standalone articles.  They can be easier to digest than broader topics, which can sometimes lack details or become too large as result of covering everything.

**Reply by annezazu** [reply]
Didn't realize feedback was waiting on me! Can you share more about what you want to see? I can't tell at a glance: https://github.com/WordPress/developer-blog-content/discussions/66

**Reply by DaisyOlsen** [reply]
Please let me know the outcome of the conversation. I'd like to get started on the draft for this, but only if it seems likely to be accepted as a stand-alone topic.

**Comment by DaisyOlsen** [comment]
Based on the conversation over on #66 I'm going to go ahead and convert this to an issue and will plan to begin writing the post. Happy for any additional discussion to happen on the issue.

**Comment by bph** [comment]
I locked the conversation. For further discussion please comment on the issue

- #81

**Comment by bph** [comment]
Published as [Block theme templates: the easy way to build an elegant grid of posts](https://developer.wordpress.org/news/2023/03/block-theme-templates-the-easy-way-to-build-an-elegant-grid-of-posts/)
